src/ucp/core/ucp_worker.c

```diff
@@ -2148,6 +2149,8 @@ ucs_status_t ucp_worker_create(ucp_context_h context,
      */
     ucs_config_parser_warn_unused_env_vars_once(context->config.env_prefix);
 
+    ucs_vfs_obj_add_dir(context, worker, "worker/%p", worker);
```

Comment:
shouldn't it be worker's name?

Reply:
https://github.com/openucx/ucx/pull/6613